src/sam/template.yaml

```diff
@@ -608,40 +617,37 @@ Resources:
         - !Ref LambdaLayer
       Policies:
         - 'arn:aws:iam::aws:policy/AmazonSSMFullAccess'
-        - 'arn:aws:iam::aws:policy/AmazonSNSFullAccess'
-        - 'arn:aws:iam::aws:policy/AmazonS3FullAccess'
-        - 'arn:aws:iam::aws:policy/AmazonCognitoReadOnly'
         - !Ref DmpHubDynamoTableReadPolicy
+        - !Ref S3BucketPolicy
       DeadLetterQueue:
         Type: 'SQS'
         TargetArn: !Ref DeadLetterQueueArn
       Environment:
         Variables:
           LAMBDA_ENV: !Ref Env
           LOG_LEVEL: !Ref DebugLevel # 'debug'
-          SNS_FATAL_ERROR_TOPIC: !Ref SnsEmailTopicArn
-          COGNITO_USER_POOL_ID: !Ref CognitoUserPoolArn
+          DMP_ID_BASE_URL: 'doi.org'
           DYNAMO_TABLE: !Ref DynamoTableName
+          SNS_FATAL_ERROR_TOPIC: !Ref SnsEmailTopicArn
           CORS_ORIGIN: !If
             - IsDev
             - "*"
             - !Sub "https://${DomainName}"
       Events:
-        GetDmps:
+        GetDmp:
+          # More info about API Event Source: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#api
           Type: 'Api'
           Properties:
             RestApiId: !Ref DmpHubRestApi
-            Path: '/dmps'
+            Path: '/dmps/{dmp_id+}'
             Method: 'get'
             Auth:
-              Authorizer: 'DmpHubCognitoAuthorizer'
-              AuthorizationScopes:
-                - !Sub 'https://auth.${DomainName}/${Env}.read'
+              Authorizer: 'NONE'
 
-  GetDmpsLogGroup:
+  GetDmpLogGroup:
     Type: 'AWS::Logs::LogGroup'
     Properties:
-      LogGroupName: !Sub "/aws/lambda/${GetDmpsFunction}"
+      LogGroupName: !Sub "/aws/lambda/${GetDmpFunction}"
```

Comment:
L650: Should "${GetDmpFunction}" be "${GetDmpsFunction}" ?

Reply:
No, that one belongs to the GetDmp function and the one above belongs to the GetDmps function. 
I must have swapped their position in the file at some point